Automattic/jetpack
# Update/my jetpack hide purchases elements when complete is active

## Proposed changes:

* Remove purchase related elements when Complete is on site
Note: We are adding AI to Complete, so no purchases should be necessary if the site has Complete active on it

### Other information:


## Jetpack product discussion

P2: pbNhbs-cbd-p2

## Does this pull request change what data or activity we track or use?

No

## Testing instructions:

1. Checkout this branch via the Jetpack Beta plugin or your local dev environment
2. Purchase Jetpack Complete on your site
3. Go to `/wp-admin/admin.php?page=my-jetpack` and ensure you do not see the "Purchase a plan" and "Activate a license" links in the My Plan section
![image](https://github.com/user-attachments/assets/b8c039da-172b-422e-a842-86865ff8dd76)
4. Now remove the license and ensure you can see them again
![image](https://github.com/user-attachments/assets/51311adb-5736-4a36-a771-9d4a0b24a9ba)
